script/prerequisitesForGetToolchains.sh

```diff
@@ -1,8 +1,8 @@
-#!/bin/sh
+#!/bin/bash
 
 # This shell script add PPA to get onecc-docker package for ONEToolchain.
 
 PPA_NAME="ppa:one-compiler/onecc-docker"
 
-add-apt-repository -y ${PPA_NAME} > /dev/null 2>&1 &&
-        apt-get update > /dev/null 2>&1
+add-apt-repository -y ${PPA_NAME}
+apt-get update -yqq
```

Comment:
AFAIK, `-yqq` is not related with printing output value, isn't it?

Reply:
My intention is to print the `add-apt-repository` logs. I thought the `apt-get update` log was too long and unnecessary.

Reply:
OK, I see it!

Reply:
Ah, `-yqq` means that no output except for errors. It does not related to output value. :)